bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/handler/VeluxBridgeHandler.java

```diff
@@ -578,6 +578,9 @@ private void syncChannelsWithProducts() {
                     logger.trace("syncChannelsWithProducts(): channel {} not found.", channelUID);
                     continue;
                 }
+                if (channelUID.getId().equals(VeluxBindingConstants.CHANNEL_ACTUATOR_LIMIT_MINIMUM) || channelUID.getId().equals(VeluxBindingConstants.CHANNEL_ACTUATOR_LIMIT_MAXIMUM)) {
```

Comment:
This is NOT the solution. It will cause those limit channels' state to never be updated in OH.

The outer loop `for (VeluxProduct product : bridgeParameters.actuators.getChannel().existingProducts.valuesOfModified())` iterates over an array of changed channels. And that array should not include the limitXXX channels, because (as you say) those channels should not have had their values modified by the respective API function call.

Reply:
@gs4711 do you have any ideas what is going wrong?